users/tasks.py

```diff
@@ -0,0 +1,11 @@
+from invoke import run, task
+
+
+@task
+def init_db(ctx):
+    pass
+
+
+@task
+def run(ctx):
```

Comment:
These tasks is empty. Why do you need an empty function?

Reply:
done

Reply:
saved for later